nix/restinio/default.nix

```
@@ -0,0 +1,47 @@
+{ lib
+, stdenv
+, fetchFromGitHub
+, cmake
+, http-parser
+, fmt
```

That is what `spdlog` uses, mixing two versions would probably fail.
```suggestion
, fmt_8
```